src/Compilers/CSharp/csc/csc.csproj

```diff
@@ -42,4 +41,5 @@
     </None>
   </ItemGroup>
   <Import Project="..\..\Core\CommandLine\CommandLine.projitems" Label="Shared" />
+  <Import Project="$(RepositoryEngineeringDir)targets\DiaSymReaderNative.targets" />
```

Comment:
❔ Does this resolve before Arcade is restored? Want to make sure it doesn't block project load when opening before a command line restore.

Reply:
This style of Import is used already in this project hence it should be no different than the behavior we have today.